src/components/account/Register.js

```diff
@@ -76,23 +76,7 @@ const Register = () => {
                     children: JSON.stringify(values.children),
                     pets: JSON.stringify(values.pets),
                   };
-                  axios
-                    .post(
-                      `${process.env.REACT_APP_BASE_URL}/auth/register`,
-                      userValues
-                    )
-                    .then((res) => {
-                      setSubmitting(false);
-                      history.push("/register-check-email");
-                    })
-                    .catch((err) => {
-                      setSubmitting(false);
-                      console.dir({
-                        err,
-                        message: err.message,
-                        stack: err.stack,
-                      });
-                    });
+                  console.log(userValues);
```

Comment:
@bvneilson: Did you intend to leave this here? Seems the Registration is effectively disabled since the axios call is removed.

Reply:
Nope! Thanks for helping me get everything in order Paul!